.github/workflows/basic.yml

```diff
@@ -0,0 +1,50 @@
+name: Java CI
```

Comment:
Wow! Greatjob implementing CI for your Duke! would be helpful in team projects later